packages-exp/auth-exp/rollup.config.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import typescriptPlugin from 'rollup-plugin-typescript2';
+import typescript from 'typescript';
+import pkg from './package.json';
+
+const deps = Object.keys(
+  Object.assign({}, pkg.peerDependencies, pkg.dependencies)
+);
+
+/**
+ * ES5 Builds
+ */
+const es5BuildPlugins = [
+  typescriptPlugin({
+    typescript
+  })
+];
+
+const es5Builds = [
+  /**
+   * Browser Builds
+   */
+  {
+    input: 'index.ts',
+    output: [
+      { file: pkg.browser, format: 'cjs', sourcemap: true },
```

Comment:
Planning to stop making `cjs` builds for the browser target in new packages. `cjs` files are not treeshakable, so they are not useful for browsers. We will only need `cjs` for the node build. This line can be deleted.

`pkg.browser` and `pkg.module` should point to the same esm build.

Reply:
Done